Library/Formula/commonmark.rb

```diff
@@ -0,0 +1,51 @@
+class Commonmark < Formula
+  url "https://github.com/jgm/CommonMark/archive/0.15.tar.gz"
+  homepage "http://commonmark.org"
+  sha1 "877665a96fdc5fcc42ec2cd605d8535344a27b72"
+
+  depends_on "cmake" => :build
+
+  def install
+    # a build directory is necessary since building from the source
+    # directory is forbidden.
+    mkdir "build"
+    Dir.chdir("build") do
```

Comment:
our `mkdir` accepts a block to run in the newly-created directory, so `mkdir "build" do` is enough